test/boulder-config.json

```diff
@@ -134,9 +134,9 @@
     "userAgent": "boulder",
     "debugAddr": "localhost:8004",
     "portConfig": {
-      "simpleHTTPPort": 5001,
-      "simpleHTTPSPort": 5001,
-      "dvsniPort": 5001
+      "httpPort": 5001,
+      "httpsPort": 5001,
+      "tlsPort": 5001
```

Comment:
it makes client-server integration testing hard; can you update it now, so that I can work on the client implementation?